core/.gas-snapshot

```diff
@@ -1,26 +1,26 @@
 BorrowerGasTest:test_addMargin() (gas: 16203)
-BorrowerGasTest:test_borrow() (gas: 97019)
-BorrowerGasTest:test_getUniswapPositions() (gas: 5057)
-BorrowerGasTest:test_modify() (gas: 68722)
-BorrowerGasTest:test_modifyWithAnte() (gas: 75180)
-BorrowerGasTest:test_repay() (gas: 98593)
-BorrowerGasTest:test_uniswapDepositInBorrower() (gas: 294787)
+BorrowerGasTest:test_borrow() (gas: 102053)
+BorrowerGasTest:test_getUniswapPositions() (gas: 5241)
+BorrowerGasTest:test_modify() (gas: 73575)
+BorrowerGasTest:test_modifyWithAnte() (gas: 80033)
+BorrowerGasTest:test_repay() (gas: 103572)
+BorrowerGasTest:test_uniswapDepositInBorrower() (gas: 299971)
 BorrowerGasTest:test_uniswapDepositStandard() (gas: 167558)
-BorrowerGasTest:test_uniswapWithdraw() (gas: 136485)
-BorrowerGasTest:test_withdraw() (gas: 123651)
-FactoryGasTest:test_createBorrower() (gas: 156320)
-FactoryGasTest:test_createMarket() (gas: 3789361)
-LenderGasTest:test_accrueInterest() (gas: 46555)
+BorrowerGasTest:test_uniswapWithdraw() (gas: 140522)
+BorrowerGasTest:test_withdraw() (gas: 128504)
```

Comment:
Increases in Borrower gas are due to the tests using proxies, as is done in prod. Previously the tests were dealing with a raw Borrower.

Jumping through proxy hoops = more gas.